### DISCUSSION TITLE
Ingress controller custome load balancer name

### DISCUSSION THREAD

**Discussion by prksh823** [nginx/kubernetes-ingress#5823] [General]
Hi
I am trying to deploy ingress nginx load balancer on aws eks cluster and controller version is 1.5.1. Its deployed successfully. Now i want to change random load balancer name to my custom name for that i am using annotation service.beta.kubernetes.io/aws-load-balancer-name: "my-clb-name". But this is not working . Can you please suggest why this is happening.

**Comment by jjngx** [comment]
Hi @prksh823 , could you please provide your deployment file? The controller version you provided (1.5.1) can suggest you are using community NGINX ingress controller (https://github.com/kubernetes/ingress-nginx), not NGINX Ingress Controller (this project).

**Comment by brianehlert** [comment]
1.5.1 of this project was released in July 2019.  This version is not compatible with current versions of Kubernetes due to changes with the K8s API.
https://github.com/nginxinc/kubernetes-ingress/releases/tag/v1.5.1
